src/utils/allocators.zig

```diff
@@ -445,6 +445,10 @@ pub const failing = struct {
     }
 };
 
+pub fn noAlloc(_: *anyopaque, _: usize, _: u8, _: usize) ?[*]u8 {
+    return null;
+}
+
```

Comment:
What's this for? Appears to be dead code, and serves a purpose redundant with `failing.allocator(.{}).vtable.alloc`.

Reply:
I forgot to remove this when I moved away from the approach that relied on it.

Reply:
2b5c2496e7685cb9a8b5eba15047cb6c1cb2aa00